OrbitCore/PathTest.cpp

```diff
@@ -96,3 +97,151 @@ TEST(Path, JoinPathPartsAbsoluteRootBackslash) {
 #endif
   EXPECT_EQ(actual, expected);
 }
+
+TEST(Path, FileNamesAndExtensions) {
+#ifdef _WIN32
+  std::string abs_input[] = {"C:\\some_abs_path\\file.txt", "C:\\dir.with.dots\\file_without_ext"};
+  std::string abs_stripped[] = {"C:\\some_abs_path\\file", "C:\\dir.with.dots\\file_without_ext"};
+#else
+  std::string abs_input[] = {"/some_abs_path/file.txt", "/dir.with.dots/file_without_ext"};
+  std::string abs_stripped[] = {"/some_abs_path/file", "/dir.with.dots/file_without_ext"};
+#endif
```

Comment:
also maybe remove this ifdefs and test for all 4 cases on both platforms?

Reply:
Right, actually that's the nicer solution. I've removed this, tests are now executed the same on both OS and should yield the same results for window-, unix- and mixed-style paths